cmd/format.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/tidwall/pretty"
+)
+
+var formatJSONCmd = &cobra.Command{
+	Use:                   "format-json [text]",
+	Short:                 "Cleans and format JSON",
+	Args:                  cobra.ExactArgs(1),
+	DisableFlagsInUseLine: true,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println(string(pretty.Pretty([]byte(args[0]))))
```

Comment:
How do you think print with color?
```suggestion
		fmt.Print(string(pretty.Color(pretty.Pretty([]byte(args[0])), nil)))
```

Reply:
Great!

I have been done for this suggestion~!